File: calc.hs
main = do
    putStrLn "pls enter your first number"
    num1 <- getLine
    putStrLn "pls enter your second number"
    num2 <- getLine
    putStrLn "pls enter your sign"
    sign <- getLine
    
    if sign == "+"
        then "pls enter your sign (again)"

File: calc.swift
print("pls enter your first number")
let num1 = Int(readLine() ?? "") ?? 0
print("pls enter your second number")
let num2 = Int(readLine() ?? "") ?? 0
print("pls enter your sign")
var sign = readLine()

if (sign == "+") {
  let result = num1 + num2
  print("your answer is \(result)")
}
if (sign == "-") {
  let result = num1 - num2
  print("your answer is \(result)")
}
if (sign == "*") {
  let result = num1 * num2
  print("your answer is \(result)")
}
if (sign == "/") {
  let result = num1 / num2
  print("your answer is \(result)")
}
// switf rating cant setup compiler 😥 / 10